indico/modules/events/client/js/reviewing/components/JudgmentModal.module.scss

```diff
@@ -0,0 +1,12 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+:global(.ui.modal) {
+    :global(.disabled.field) {
+        background: none;
```

Comment:
why do we need this?

Reply:
Because `disabled` fields have some color that really stands out when used in conjunction with out `i-timeline` styles